backend/src/models/Settings.js

```diff
@@ -0,0 +1,9 @@
+import mongoose from 'mongoose';
+
+const settingsSchema = new mongoose.Schema({
+  appTheme: { type: String, enum: ['light', 'dark'], default: 'light' },
+  soonExpLenght: { type: Number, enum: [3, 5, 7], default: 3 },
```

Comment:
I think there is a typo here Lenght -> Length.

Also to make it more understandable let's call it soonExpiringFilterLength.